package.json

```
@@ -3,27 +3,38 @@
   "version": "1.0.0",
   "description": "webpack and configuration",
   "main": "index.js",
+  "jest": {
+    "coveragePathIgnorePatterns": [
+      "/index.js"
+    ],
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
+      "\\.(css|less)$": "identity-obj-proxy"
+    }
+  },
   "scripts": {
     "start:dev": "NODE_ENV=development webpack-dev-server --open",
     "start:prod": "NODE_ENV=production webpack-dev-server --open",
-    "test": "jest",
+    "test": "jest ./tests/ --coverage",
```

You can remove **coverage** from test command because you already have separate command for this
```suggestion
    "test": "jest",
```